From b03c3c820949d9cf4bbc0c45b4bcb383f9c63d34 Mon Sep 17 00:00:00 2001
From: Xie Xiaobo <X.Xie@freescale.com>
Date: Sun, 26 Apr 2009 17:18:50 +0800
Subject: [PATCH] ENGR00111675 Fixed GPIO pins wrong for i.MX25 ESDHC

Inactive the ESDHC GPIO pin have issue, it would resulted in kernel dump.

Signed-off-by: Xie Xiaobo <X.Xie@freescale.com>
---
 arch/arm/mach-mx25/mx25_3stack_gpio.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx25/mx25_3stack_gpio.c b/arch/arm/mach-mx25/mx25_3stack_gpio.c
index 422bdc5..ec6f2f3 100644
--- a/arch/arm/mach-mx25/mx25_3stack_gpio.c
+++ b/arch/arm/mach-mx25/mx25_3stack_gpio.c
@@ -854,6 +854,8 @@ void gpio_sdhc_inactive(int module)
 		mxc_free_iomux(MX25_PIN_SD1_DATA1, MUX_CONFIG_GPIO);
 		mxc_free_iomux(MX25_PIN_SD1_DATA2, MUX_CONFIG_GPIO);
 		mxc_free_iomux(MX25_PIN_SD1_DATA3, MUX_CONFIG_GPIO);
+		mxc_free_iomux(MX25_PIN_A14, MUX_CONFIG_GPIO);
+		mxc_free_iomux(MX25_PIN_A15, MUX_CONFIG_GPIO);
 		break;
 	case 1:
 		/* SDHC2 */
-- 
1.5.4.4

